### PR TITLE
Issue 330 - direct users to portal in email alert about new join requests 

### PR DESCRIPTION
**Pull request by jofeinstein**
Added a link to the email sent to PIs/managers that contains a link to approve/deny join requests. Also changed the wording of the email from "via MyBRC user portal" to "via the MyBRC User Portal".

### CODE REVIEW

File: coldfront/core/project/utils.py [hunk 1]
```diff
@@ -162,7 +160,20 @@ def send_project_join_notification_email(project, project_user):
             'user__email', flat=True
         ))
 
-    send_email_template(subject, template_name, context, sender, receiver_list)
+    msg_plain = \
```
Comment (matthew-li) [inline coldfront/core/project/utils.py:163]: Thanks, not rendering links directly is nice.

In #309/#310, we'll want to come up with a uniform way of sending HTML emails.

File: coldfront/templates/email/new_project_join_request.txt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Dear managers of {{ project_name }},
 
-User {{ user_string }} has requested to join your project, {{ project_name }} via MyBRC user portal. Please approve/deny this request.
+User {{ user_string }} has requested to join your project, {{ project_name }} via the MyBRC User Portal.
+
+Please approve/deny this request here.
```
Comment (matthew-li) [inline coldfront/templates/email/new_project_join_request.txt:5]: In the (unlikely?) scenario that the user's browser doesn't support HTML emails, it'll fall back to this `.txt`, which won't actually have the URL. We should include it:

```
...this request here: {{ review_url }}.
```

(I imagine there are places in the code where this isn't the case, but we'll resolve those in #309.)

**Review by matthew-li — CHANGES_REQUESTED** [review]
LGTM, thanks! I had one minor comment.

File: coldfront/templates/email/new_project_join_request.txt (hunk 2 above)
Comment (jofeinstein) [reply coldfront/templates/email/new_project_join_request.txt]: made the change

**Pull request merged**